addons/test_mail/tests/test_message_post.py

```diff
@@ -36,6 +41,19 @@ def test_message_post_return(self):
         messageId = call_kw(self.env['mail.channel'], 'message_post', [test_channel.id], {'body': 'test'})
         self.assertTrue(isinstance(messageId, int))
 
+    @users('demo')
```

Comment:
Use "employee" instead of demo (crafted for mail tests while demo may be changed by people in backend)